osmaxx/conversion/management/commands/result_harvester.py

```diff
@@ -41,6 +41,10 @@ def _handle_running_jobs(self):
             self._update_job(job_id=job_id, queue=queue)
 
     def _update_job(self, job_id, queue):
+        if job_id is None:
+            logger.exception("job_id is None, None is not a valid id!")
+            return
```

Comment:
`_update_job` is called in loops in `_handle_failed_jobs()` and `_handle_running_jobs()`. Should the loop bodies be wrapped with `try`, so that one `None` in the job IDs doesn't keep all jobs after it in the list from being updated?